src/views/SavedItems.vue

```diff
@@ -0,0 +1,49 @@
+
+
+<template>
+  <UpperBar></UpperBar>
+  <div>
+    <b-card
+        header="Saved Items"
+        header-text-variant="white"
+        header-tag="header"
+        header-bg-variant="success"
+        footer-border-variant="dark"
+    >
+    <b-card-body>
+        <b-card-group deck>
+            <ItemCard></ItemCard>
+            <ItemCard></ItemCard>
+            <ItemCard></ItemCard>
+        </b-card-group>
+    </b-card-body>
+  </b-card>
+  </div>
+</template>
+
+<script>
+  import ItemCard from 'C:/Compsci WIP/Capstone/capstoneProj/Store-Website/src/components/ItemCard.vue'
+  import UpperBar from 'C:/Compsci WIP/Capstone/capstoneProj/Store-Website/src/components/UpperBar.vue'
```

Comment:
This won't work for the rest of the team, maybe make the imports relative to the project folder?

Reply:
Fixed this in recent commit